### ISSUE TITLE
Jak And Daxter : The Lost Frontier video sound break

### ISSUE THREAD

**Issue by zminhquanz** [hrydgard/ppsspp#9310]
![image](https://cloud.githubusercontent.com/assets/11285640/22958698/82b6ba36-f363-11e6-8fb9-a0187d993919.png)

**Comment by hrydgard** [comment]
Hm, if the CPU is executing the BREAK instruction, then something bad has happened.

I guess the whole thing freezes at this point?

**Comment by zminhquanz** [comment]
@hrydgard yes , it's work before , when it's go to cutscene , it's break

**Comment by hrydgard** [comment]
No, i mean, did it work in earlier builds of PPSSPP? If so, when did it break?

**Comment by zminhquanz** [comment]
it show in picture , build 1.3 official

**Comment by LunaMoo** [comment]
By version which broke it, Henrik is asking for something more specific - Which was the first build that reproduces the issue.
 It can't be 1.3 since release versions are just a number change, so use [buildbot](http://buildbot.orphis.net/ppsspp/index.php?m=fulllist) and check builds around half way between what you know is broken and one that you remember working, and continue checking until you find a change which caused the problem.

 Could be related to #8991

**Comment by zminhquanz** [comment]
@LunaMoo yeah , it's break and freeze/hang in this time when you play cutscene

**Comment by marosis** [comment]
Zninhquanz FACEPALM

**Comment by Emulatorer** [comment]
This game is running fine to my device Mali-450 GPU 1.4GHz CPU Octacore using latest beta build 15/30 90%-100% Speed

I use skipp-buffered rendering the sound is really clear but sometimes flickering 😅

When set to buffered rendering the game is so slow even if I use x1 frameskipping I need to enable the "disable slower effects"
but skipp-buffered rendering  still better

Hoping this game will improve in ppsspp v1.9.0 🙂

**Comment by Emulatorer** [comment]
I notice the annoying crackling sound when set to buffered rendering

but the sojnd in skipp-buffered rendering is ok (imo) 
the only issue I encounter is when you hit the brown mushrooms the screens goes black except for the map for about 1-2 seconds 🤔

**Comment by Panderner** [comment]
This is happens when I turned on frame skip, I set 1 to 8 this happens.

**Comment by Gamemulatorer** [comment]
Can't reproduce this.

**Comment by unknownbrackets** [comment]
By "this happens", do you mean the BREAK error in the log / game freeze, or do you mean the screen going black for 1-2 seconds after hitting the mushrooms?

For the mushroom thing, a GE frame dump might help (but I realize it might be hard to catch if it's only for 1-2 seconds...)

-[Unknown]

**Comment by Gamemulatorer** [comment]
> By "this happens", do you mean the BREAK error in the log / game freeze, or do you mean the screen going black for 1-2 seconds after hitting the mushrooms?
> 
> For the mushroom thing, a GE frame dump might help (but I realize it might be hard to catch if it's only for 1-2 seconds...)
> 
> -[Unknown]

No! I mean is I cannot reproduce his issue, about the mushrooms after hitting is yes can still reproduce the issue but because I'm using skip-buffered rendering but not on buffered rendering which is the default.

**Comment by Gamemulatorer** [comment]
@zminhquanz feedback?
